src/System.Xml.XPath.XDocument.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30723.0
+# Visual Studio 14
+VisualStudioVersion = 14.0.22310.1
```

Comment:
VS is gonna keep changing that every time I will change a solution. Since we cannot run tests on the older VS we need to use VS 2015 anyway. I'm ok with either keeping and removing the change. What are your thoughts?